apm-sniffer/bootstrap-plugins/jdk-threadpool-plugin/src/main/java/org/apache/skywalking/apm/plugin/ThreadPoolExecuteMethodInterceptor.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package org.apache.skywalking.apm.plugin;
+
+import org.apache.skywalking.apm.plugin.wrapper.SwRunnableWrapper;
+import org.apache.skywalking.apm.agent.core.context.ContextManager;
+import java.util.concurrent.RunnableFuture;
+
+public class ThreadPoolExecuteMethodInterceptor extends AbstractThreadingPoolInterceptor {
+
+    @Override
+    public Object wrap(Object param) {
+        if (param instanceof SwRunnableWrapper) {
+            return param;
+        }
+
+        if (param instanceof RunnableFuture) {
+            return null;
```

Comment:
Would not `return null` cause an NPE? Same as below.

Reply:
it won't cause NPE. Let's see the caller code in 'beforeMethod'.
```
        Object updateResult = wrap(argument);
        if (updateResult != null) {
            allArguments[0] = updateResult;
        }
```

Reply:
> OK, please add comments on `AbstractThreadingPoolInterceptor#wrap`.
> 
> Also consider to add more comments on codes too.
Ok!